lang/cs/Org.Apache.REEF.Wake.Tests/TransportTest.cs

```diff
@@ -213,5 +216,16 @@ private static ITcpPortProvider GetTcpProvider(int portRangeStart, int portRange
                 .Build();
             return TangFactory.GetTang().NewInjector(configuration).GetInstance<ITcpPortProvider>();
         }
+
+        private static ITcpClientFactory GetTcpClientFactory(int connectionRetryCount, int sleepTimeInMs)
```

Comment:
I do not want to create a separate file for it especially....Hence would keep it as it is ... Again as above configuration module involves providing Driver configurations and all and Wake and hence its tests are independent of it.